code/modules/events/portal_storm.dm

```diff
@@ -21,6 +21,45 @@
 	hostile_types = list(/mob/living/simple_animal/hostile/construct/armored/hostile = 8,\
 						/mob/living/simple_animal/hostile/construct/wraith/hostile = 6)
 
+/datum/round_event_control/portal_storm_mesa
+	name = "Portal Storm: Mesa"
+	typepath = /datum/round_event/portal_storm/portal_storm_mesa
+	weight = 0
+	max_occurrences = 0
+
+/datum/round_event/portal_storm/portal_storm_mesa
+	boss_types = list(/mob/living/simple_animal/hostile/jungle/leaper = 4, \
+						/mob/living/simple_animal/hostile/jungle/mega_arachnid = 6, \
+						/mob/living/simple_animal/hostile/asteroid/hivelord = 5)
+	hostile_types = list(/mob/living/carbon/alien/larva = 2, \
```

Comment:
larvas dont move

Reply:
it's mostly for fluff, I wanted to spook people a bit

Reply:
The bad part here is that some funny (TM) xenobiologist will give it a sentience potion.